src/main/java/com/revealprecision/revealserver/api/v1/facade/controller/TaskFacadeController.java

```diff
@@ -18,11 +20,12 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/rest")
+@RequestMapping("/rest/task/v2")
```

Comment:
android currently has endpoints **/rest/v2/task/sync**, **/rest/v2/task/add** and **/rest/v2/task/update_status** set. So if you rename we can decide to accept if we agree we will make change on the android?

Reply:
this was an error on my side